### PR TITLE
Add page titles to filesystem routes

### PR DESCRIPTION
**Pull request by backspace**
I wish something like Ruby’s `Pathname` were a trivial inclusion 😐

### CODE REVIEW

File: ui/app/controllers/allocations/allocation/task/fs.js [hunk 1]
```diff
@@ -20,6 +20,16 @@ export default Controller.extend({
   directories: filterBy('directoryEntries', 'IsDir'),
   files: filterBy('directoryEntries', 'IsDir', false),
 
+  pathWithLeadingSlash: computed('path', function() {
+    const path = this.path;
+
+    if (path.startsWith('/')) {
+      return path;
+    } else {
+      return `/${path}`;
```
Comment (endocrimes) [inline ui/app/controllers/allocations/allocation/task/fs.js]: We might also want to check that it doesn't start with `[A-Z]:\` too? Not sure how the FS api works on windows tho tbh

Comment (backspace) [reply ui/app/controllers/allocations/allocation/task/fs.js]: hmm good question, thanks. Is there some cloud-y deployment of Nomad I can use? Or maybe I can set it up locally in a virtual machine 😯

Comment (endocrimes) [reply ui/app/controllers/allocations/allocation/task/fs.js]: A local vm is probably your best bet right now :disappointed:

Comment (backspace) [reply ui/app/controllers/allocations/allocation/task/fs.js]: I finally managed it. It looks like `\`s are abstracted away:

image

I was able to query `server\empty.txt` and get the same response. So maybe I don’t have to care about `\`s 🤔

Comment (endocrimes) [reply ui/app/controllers/allocations/allocation/task/fs.js]: :tada:

**Pull request merged**